### PR TITLE
OCA rebranding + Documentation updated

### PR DESCRIPTION
**Pull request by sebalix**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -71,23 +72,23 @@ How does it work? See below:
 See the documentation for more details and features.
 
 Supported Odoo server versions
-------------------------------
+==============================
 
 `OdooRPC` has been tested on `Odoo` 8.0, 9.0 and 10.0.
```
Comment (pedrobaeza) [inline README.rst]: Include 11.0

Comment (sebalix) [reply README.rst]: Officially the 11.0 is not supported yet, I'll make another PR for that (to test it on Travis + updating the documentation). The support of Python 3.2 and 3.3 will be dropped too.

Comment (pedrobaeza) [reply README.rst]: OK then

**Comment by sebalix** [comment]
About PR approvals, the standard OCA rules apply?

**Comment by pedrobaeza** [comment]
Well, that are the general rules for OCA repositories, but an specific PSC can decide others.

**Comment by sebalix** [comment]
I would like to merge this ASAP to continue the work on v11 and publish a new release on PyPI. You think we can go ahead?

**Comment by pedrobaeza** [comment]
There are not critical changes, so I merge.

**Pull request merged**